sdk/eventhub/event-hubs/test/hubruntime.spec.ts

```diff
@@ -35,7 +35,7 @@ describe("RuntimeInformation", function(): void {
   });
 
   function arrayOfIncreasingNumbersFromZero(length: any): Array<string> {
-    return Array.apply(undefined, new Array(length)).map((x: any, i: any) => {
+    return Array.apply(undefined, new Array(length)).map((_x: any, i: any) => {
```

Comment:
nit: Array.fill() is nicer, but not sure about the min version requirements of using it...

Reply:
I love your code reviews! They're so detailed. Among my favorite code reviews ever ♥

Reply:
This code:

```ts
    return Array.apply(undefined, new Array(length)).map((_x: any, i: any) => {
      return `${i}`;
    })
```

Is so unnecessarily complicated!

Here's an alternative:

```ts
    const result = new Array(length);
    for (let i = 0; i < length; i++) {
      result[i] = `${i}`;
    }
    return result;
```

Let me know what you think!

Reply:
That also works and is a lot easier to understand.